CLDSRV-527

KazToozs opened the pull request:
# Pull request template

## Description

### Motivation and context

Why is this change required? What problem does it solve?

### Related issues

Please use the following link syntaxes #600 to reference issues in the
current repository

## Checklist

### Add tests to cover the changes

New tests added or existing tests modified to cover all changes

### Code conforms with the [style guide](https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#coding-style-guidelines)

### Sign your work

In order to contribute to the project, you must sign your work
https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

Thank you again for contributing! We will try to test and integrate the change
as soon as we can.

bert-e commented:
# Hello kaztoozs,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/cloudserver/doc/user).





<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-527 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `7.70.47`

* `8.6.27`

* `8.7.48`

* `8.8.21`


Please check the `Fix Version/s` of CLDSRV-527, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-527 contains:


* `7.70.47`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `7.70.48`

* `8.6.27`

* `8.7.48`

* `8.8.25`


Please check the `Fix Version/s` of CLDSRV-527, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-527 contains:


* `7.70.47`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `7.70.48`

* `8.6.27`

* `8.7.48`

* `8.8.26`


Please check the `Fix Version/s` of CLDSRV-527, or the target
branch of this pull request.

Commit:
[edit]CLDSRV-527: add necessary packages

Commit:
CLDSRV-527: accomodate POST obj API call

- Use busboy to separate out form data fields
- Handle POST Object separately from other POST API calls
- Update the API call waterfall with this new logic

Commit:
CLDSRV-527: add POST Object API Handler

Commit:
CLDSRV-527: accomodate POST Object in extraneous code

Commit:
[WIP]CLDSRV-527: add test files

Commit:
fixup: packages

BourgoisMickael commented on tests/functional/aws-node-sdk/test/object/post.js:
this tests put object, not post I think

Commit:
first functional tests are fucntional

Commit:
update package

Commit:
update func tests

Commit:
fix other posts

Commit:
checkpoint: changing the parser

Commit:
checkpoint: parser changed to custom, very slow on large files

Commit:
checkpoint: acl tests

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-527 contains:


* `7.70.47`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `7.70.48`

* `8.6.27`

* `8.7.48`

* `8.8.27`


Please check the `Fix Version/s` of CLDSRV-527, or the target
branch of this pull request.

Commit:
restablish busboy

Commit:
quick fixes + unit tests
